docs/GettingStartedDocs/Contributors/ExperimentalSupportRHEL8.md

```diff
@@ -0,0 +1,122 @@
+
+```bash
+sudo scripts/ansible/install-ansible.sh
+```
+
+On a system which supports SGX1 or SGX+FLC, use the command below to set up prerequisites. Please note that the Open Enclave SDK does not support attestation on RHEL as yet.
+
+```bash
+ansible-playbook scripts/ansible/oe-contributors-setup-sgx1.yml
+```
+
+On a system which does not support SGX1 or SGX1+FLC, use the command below to set up prerequisites. You would need to run tests in simulation mode.
+
+```bash
+ansible-playbook scripts/ansible/oe-contributors-setup.yml
+```
+
+## Build
+
+To build first create a build directory ("build/" in the example below) and change into it.
+
+```bash
+export CC=gcc
+export CXX=g++
+mkdir build/
+cd build/
+```
+
+Then run `cmake` to configure the build and generate the make files and build:
+
+```bash
+cmake -DHAS_QUOTE_PROVIDER=OFF ..
+make
+ctest
+```
+
+## Run unit tests
+
+After building, run all unit test cases using `ctest` to confirm the SDK is built and working as expected.
+
+Run the following command from the build directory if your system supports SGX1 or SGX1+FLC:
+
+```bash
+ctest
+```
+
+If your system does not have SGX support, run the following:
+
+```bash
+OE_SIMULATION=1 ctest
+```
+
+You will see test logs similar to the following:
+
+```bash
+Test project /home/youradminusername/openenclave/build
+      Start   1: tests/aesm
+1/123 Test   #1: tests/aesm ...............................................................................................................   Passed    0.98 sec
+      Start   2: tests/mem
+2/123 Test   #2: tests/mem ................................................................................................................   Passed    0.00 sec
+      Start   3: tests/str
+3/123 Test   #3: tests/str ................................................................................................................   Passed    0.00 sec
+....
+....
+....
+122/123 Test #122: tools/oedump .............................................................................................................   Passed    0.00 sec
+            Start 123: oeelf
+123/123 Test #123: oeelf ....................................................................................................................   Passed    0.00 sec
+
+100% tests passed, 0 tests failed out of 123
+
+Total Test time (real) =  83.61 sec
+```
+
+A clean pass of the above unit tests is an indication that your Open Enclave setup was successful.
+
+You can start playing with the Open Enclave samples after following the instructions in the "Install" section below to configure samples for building,
+
+For more information refer to the [Advanced Test Info](AdvancedTestInfo.md) document.
+
+## Install
+
+ Follow the instructions in the [Install Info](LinuxInstallInfo.md) document to install the Open Enclave SDK built above.
+
+## Build and run samples
+
+To build and run the samples, please look at [README_Linux.md](/samples/README_Linux.md).
```

Comment:
I think my biggest open question around the documentation of the experimental RHEL support is whether the sample instructions work correctly on RHEL. Given the issues with compiler version differences we encountered with Ubunutu 1604 vs 1804, it would be nice to ensure that someone has verified and updated that specifically.

Reply:
@CodeMonkeyLeet Thanks, I am verifying all these steps.

Reply:
On a system not support SGX/FLC, the sample helloworld and file-encryptor can only pass in this way: make && make simulate, which is not documented in README_Linux.md.

Reply:
The sample readmes covered this. I also added text to the switchless sample.